flare/framework/tests/test_computation_task.py

```diff
@@ -89,17 +89,15 @@ def test(input, ct, max):
             width=84, height=84, num_actions=num_actions))
 
         q = SimpleQ(model=SimpleModelQ(
-            dims=dims,
+            dims=[dims],
             num_actions=num_actions,
-            mlp=nn.Sequential(
+            perception_net=nn.Sequential(
```

Comment:
Why removing the last layer?

Reply:
the last layer is moved to policy_net or value_net inside SimpleModelQ